modules/gateways/callback/blockonomics.php

```diff
@@ -116,7 +110,9 @@
  */
 
 if ($blockonomics->checkIfTransactionExists($blockonomics_currency_code . ' - ' . $txid)) {
-    exit();
+    $error = 'Transaction ID is duplicated';
+    logTransaction('blockonomics', $_GET, $error);
```

Comment:
Even the bad transactions must be logged, this same logic applies in other gateways modules.